springboard/templates/base.jinja2

```diff
@@ -14,17 +14,19 @@
     <div id="header">
         <div class="lang">
             <div>
-                {% for code, name in languages %}
-                <span>
-                    <a href="/locale/?language={{code}}">{{name}}</a>
-                    {% if not forloop.last %}
-                        <span> |</span>
-                    {% endif %}
-                </span>
+                <li>
+                    <a>{{language|display_language_name}}</a>
+                </li>
+                {% for lang_code in display_languages %}
+                <li>
```

Comment:
We probably want to skip `lang_code` here if it's the same as `language` because otherwise we'd potentially get it twice.